### ISSUE TITLE
Deleted events re-appear

### ISSUE THREAD

**Issue by Trefex** [inovex/CalendarSync#167]
Hi,

When a synced event in a sink is deleted, it will be re-added on the next sync.

Could have a local store that keeps track of this and does not re-add deleted events in the sink?

Many thanks
T

**Comment by alxndr13** [comment]
Hi! We thought about this, but wouldn't it make more sense to delete the event in the source calendar then?

**Comment by Trefex** [comment]
Not necessarily, because in my use case, i'd like the target calendar for my company to be up to date with my availabilities but the source calendar i would not decline / delete events as they might still be relevant eg when i would want to send somebody else or follow up on meetings that happened without me. These are mostly meetings with 100+ people.
